Plots/Contours/NCL_conwomap_1.py

```diff
@@ -0,0 +1,63 @@
+"""
+NCL_conwomap_1.py
+=================
+This script illustrates the following concepts:
+   - Drawing a simple line contour plot
+   - Drawing a line contour plot with multiple colors
+   - Increasing the thickness of contour lines
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/conwomap_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/conwomap_1_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
```

Comment:
Probably no need for Cartopy in this example as all the projections could be simply removed (Please see my following comments for this).

Reply:
Removed cartopy in latest commit.